content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,408 @@
+                 csi:
+                   driver: "csi.spiffe.io"
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        enabled: true
+        label:
+          istio: ingressgateway
+        k8s:
+          overlays:
+            - apiVersion: apps/v1
+              kind: Deployment
+              name: istio-ingressgateway
+              patches:
+                - path: spec.template.spec.volumes.[name:workload-socket]
+                  value:
+                    name: workload-socket
+                    csi:
+                      driver: "csi.spiffe.io"
+                - path: spec.template.spec.containers.[name:istio-proxy].volumeMounts.[name:workload-socket]
+                  value:
+                    name: workload-socket
+                    mountPath: "/run/secrets/workload-spiffe-uds"
+                    readOnly: true
+EOF
+{{< /text >}}
+
+3. Use [Automatic sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
+to automatically inject the `spire` template, see [custom templates](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental),
+in the configuration above into new workloads pods, or add the pod annotation to your deployment spec:
+
+{{< text yaml >}}
+podAnnotations:
+  inject.istio.io/templates: "sidecar,spire"
+{{< /text >}}
+
+This will share the spiffe-csi-driver with the Ingress Gateway and the sidecars that are going to be automatically injected on workload pods,
+granting them access to the SPIRE Agent's UNIX Domain Socket.
+
+Check istio-ingressgateway pod state:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-5b45864fd4-lgrxs   0/1     Running   0          17s
+istiod-989f54d9c-sg7sn                  1/1     Running   0          23s
+{{< /text >}}
+
+Data plane containers will only reach the `Ready` if a corresponding registration entry is created for them on the SPIRE Server. Then,
+Envoy will be able to fetch cryptographic identities from SPIRE.
+See [Register workloads](#register-workloads) to register entries for services in your mesh.
+
+## Register workloads
+
+This section describes the options available for registering workloads in SPIRE Server.
+
+### Option 1: Automatic registration using the SPIRE workload registrar
+
+By deploying [SPIRE Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar)
+along with SPIRE Server, new entries are automatically registered for each new pod that is created.
+
+See [Verifying that identities were created for workloads](#verifying-that-identities-were-created-for-workloads) 
+to check issued identities.
+
+### Option 2: Manual Registration
+
+SPIRE Server is able to verify a group of selector values to improve workload attestation security robustness.

```

Comment:
What do you mean by SPIRE Server verifying a group of selector values ?

Reply:
I mean the selectors for the attestation

Reply:
Yeah, maybe we could elaborate bit further here?

```suggestion
To improve workload security robustness, the SPIRE Agent is able to verify agaist a group of selector values based on different attestation parameters.  
```